napari/_qt/qt_main_window.py

```diff
@@ -1179,10 +1179,10 @@ def _status_changed(self, event):
         else:
             status_info = event.value
             self._status_bar.setStatusText(
-                layer_base=status_info['layer_base'],
-                source_type=status_info['source_type'],
-                plugin=status_info['plugin'],
-                coordinates=status_info['coordinates'],
+                layer_base=status_info.get('layer_base', None),
+                source_type=status_info.get('source_type', None),
+                plugin=status_info.get('plugin', None),
+                coordinates=status_info.get('coordinates', None),
```

Comment:
This change was a bug fix, needed because `status_info` can be both a string or a dict. This is not a change from this PR, but an intended behaviour from before, see the definition of this field in `VieweModel` on `main`:
```py
    status: Union[str, Dict] = 'Ready'
```

For some reason, this issue never came up until now.

Reply:
but str does not have the method `get`. 

And does not come up as a case of the string being done by the previous branch, and the place where Dict is provided provides all keys.

Reply:
Uhm, you're right... I think I was getting confused with `getattr` and thinking I could get around it like this. Not sure what the issue is then, I need to dig deeper.